### ISSUE TITLE
Program exits starting episode 1 run

### ISSUE THREAD

**Issue by myndtrust** [IBM/rl-testbed-for-energyplus#17]
I am running the program without any modifications. I had a successful run right after install, but then undated to Ubuntu 18.0.02 from 18.04.01 and the behavior of the program changed. I have cloned the repo in another directory by following the installation instruction, but the problem persists. 

I am running the program with `time python3 -m baselines_energyplus.trpo_mpi.run_energyplus --num-timesteps 10
`
The error file also does not contain much information neither. The error file for the 1 episode is `Program Version,EnergyPlus, Version 8.8.0-7c3bbe4830, YMD=2019.05.01 19:17,IDD_Version 8.8.0
   ************* Beginning Zone Sizing Calculations`

Episode 0 had a more detailed err file, but it appears irrelevant to the exist when starting episode 1.

`Program Version,EnergyPlus, Version 8.8.0-7c3bbe4830, YMD=2019.05.01 19:09,IDD_Version 8.8.0
   ************* Beginning Zone Sizing Calculations
   ** Warning ** Calculated design heating load for zone=WEST ZONE is zero.
   **   ~~~   ** Check Sizing:Zone and ZoneControl:Thermostat inputs.
   ** Warning ** Calculated design heating load for zone=EAST ZONE is zero.
   **   ~~~   ** Check Sizing:Zone and ZoneControl:Thermostat inputs.
   ************* Beginning System Sizing Calculations
   ** Warning ** ManageSizing: Calculated Heating Design Air Flow Rate for System=WEST ZONE AIR SYSTEM is zero.
   **   ~~~   ** Check Sizing:Zone and ZoneControl:Thermostat inputs.
   ** Warning ** ManageSizing: Calculated Heating Design Air Flow Rate for System=EAST ZONE AIR SYSTEM is zero.
   **   ~~~   ** Check Sizing:Zone and ZoneControl:Thermostat inputs.
   ************* Beginning Plant Sizing Calculations
   ************* Testing Individual Branch Integrity
   ************* All Branches passed integrity testing
   ************* Testing Individual Supply Air Path Integrity
   ************* All Supply Air Paths passed integrity testing
   ************* Testing Individual Return Air Path Integrity
   ************* All Return Air Paths passed integrity testing
   ************* No node connection errors were found.
   ************* Beginning Simulation
   ** Severe  ** CalcIndirectResearchSpecialEvapCooler: calculate secondary air mass flow failed for Indirect Evaporative Cooler Research Special = EAST DATA CENTER IEC
   **   ~~~   **  Environment=WHOLEYEARDAY, at Simulation time=01/01 17:42 - 17:43
   **   ~~~   **   Iteration limit [1000] exceeded in calculating secondary air mass flow rate
   **   ~~~   **   Simulation continues
   ** Severe  ** CalcIndirectResearchSpecialEvapCooler: calculate secondary air mass flow failed for Indirect Evaporative Cooler Research Special = WEST DATA CENTER IEC
   **   ~~~   **  Environment=WHOLEYEARDAY, at Simulation time=01/01 19:38 - 19:39
   **   ~~~   **   Iteration limit [1000] exceeded in calculating secondary air mass flow rate
   **   ~~~   **   Simulation continues
   ** Warning ** CalcDoe2DXCoil: Coil:Cooling:DX:SingleSpeed="WEST DX COOLING COIL" - Air volume flow rate per watt of rated total cooling capacity is out of range at 1.100E-005 m3/s/W.
   **   ~~~   **  Environment=WHOLEYEARDAY, at Simulation time=01/01 19:38 - 19:39
   **   ~~~   ** Expected range for VolumeFlowPerRatedTotalCapacity=[2.684E-005--6.713E-005]
   **   ~~~   ** Possible causes include inconsistent air flow rates in system components,
   **   ~~~   ** or variable air volume [VAV] system using incorrect coil type.
   ** Warning ** CalcDoe2DXCoil: Coil:Cooling:DX:SingleSpeed "WEST DX COOLING COIL" - Air-cooled condenser inlet dry-bulb temperature below 0 C. Outdoor dry-bulb temperature = -1.38
   **   ~~~   **  ... Occurrence info = WHOLEYEARDAY, 01/01 19:38 - 19:39
   **   ~~~   ** ... Operation at low ambient temperatures may require special performance curves.
   ** Warning ** CalcDoe2DXCoil: Coil:Cooling:DX:SingleSpeed "WEST DX COOLING COIL" - Full load outlet air dry-bulb temperature < 2C. This indicates the possibility of coil frost/freeze. Outlet temperature = -37.94 C.
   **   ~~~   **  ...Occurrence info = WHOLEYEARDAY, 01/01 19:38 - 19:39
   **   ~~~   ** ... Possible reasons for low outlet air dry-bulb temperatures are: This DX coil
   **   ~~~   **    1) may have a low inlet air dry-bulb temperature. Inlet air temperature = 10.263 C.
   **   ~~~   **    2) may have a low air flow rate per watt of cooling capacity. Check inputs.
   **   ~~~   **    3) is used as part of a HX assisted cooling coil which uses a high sensible effectiveness. Check inputs.
   ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Tower range temperature is outside model boundaries at 2.06E-002.
   **   ~~~   **  ...Valid range = 1.11 to 22.22. Occurrence info = WHOLEYEARDAY, 01/07 20:17 - 20:18
   **   ~~~   **  ...Tower range temperature passed to the model = 2.05947E-002
   **   ~~~   **  ...Range temperatures outside model boundaries may not adversely affect tower performance.
   **   ~~~   **  ...This is not an unexpected occurrence when simulating actual conditions.
   ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Tower approach temperature is outside model boundaries at 0.44.
   **   ~~~   **  ...Valid range = 1.11 to 40.00. Occurrence info = WHOLEYEARDAY, 02/23 07:48 - 07:49
   **   ~~~   **  ...Tower approach temperature passed to the model = 0.43596
   **   ~~~   **  ...Approach temperatures outside model boundaries may not adversely affect tower performance.
   **   ~~~   **  ...This is not an unexpected occurrence when simulating actual conditions.
   ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Inlet air wet-bulb temperature is outside model boundaries at 26.68.
   **   ~~~   **  ...Valid range = -34.40 to 26.67. Occurrence info = WHOLEYEARDAY, 07/19 16:15 - 16:16
   **   ~~~   **  ...Inlet air wet-bulb temperature passed to the model = 26.666700
   **   ~~~   **  ...Wet-bulb temperatures outside model boundaries may not adversely affect tower performance.
   ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Inlet air wet-bulb temperature is outside model boundaries at 26.68.
   **   ~~~   **  ...Valid range = -34.40 to 26.67. Occurrence info = WHOLEYEARDAY, 07/19 16:17 - 16:18
   **   ~~~   **  ...Inlet air wet-bulb temperature passed to the model = 26.666700
   **   ~~~   **  ...Wet-bulb temperatures outside model boundaries may not adversely affect tower performance.
   ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Inlet air wet-bulb temperature is outside model boundaries at 26.68.
   **   ~~~   **  ...Valid range = -34.40 to 26.67. Occurrence info = WHOLEYEARDAY, 07/19 16:18 - 16:19
   **   ~~~   **  ...Inlet air wet-bulb temperature passed to the model = 26.666700
   **   ~~~   **  ...Wet-bulb temperatures outside model boundaries may not adversely affect tower performance.
   ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Inlet air wet-bulb temperature is outside model boundaries at 26.68.
   **   ~~~   **  ...Valid range = -34.40 to 26.67. Occurrence info = WHOLEYEARDAY, 07/19 16:20 - 16:21
   **   ~~~   **  ...Inlet air wet-bulb temperature passed to the model = 26.666700
   **   ~~~   **  ...Wet-bulb temperatures outside model boundaries may not adversely affect tower performance.
   ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Inlet air wet-bulb temperature is outside model boundaries at 26.68.
   **   ~~~   **  ...Valid range = -34.40 to 26.67. Occurrence info = WHOLEYEARDAY, 07/19 16:21 - 16:22
   **   ~~~   **  ...Inlet air wet-bulb temperature passed to the model = 26.666700
   **   ~~~   **  ...Wet-bulb temperatures outside model boundaries may not adversely affect tower performance.
   ************* Simulation Error Summary *************
   ************* There are 1 unused schedules in input.
   ************* Use Output:Diagnostics,DisplayUnusedSchedules; to see them.
   *************
   ************* ===== Recurring Error Summary =====
   ************* The following recurring error messages occurred.
   *************
   *************  ** Warning ** Secondary air mass flow Iteration limit exceeded in Indirect Evaporative Cooler Research Special = EAST DATA CENTER IEC
   *************  **   ~~~   **   This error occurred 4 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************
   *************  ** Warning ** Secondary air mass flow Iteration limit exceeded in Indirect Evaporative Cooler Research Special = WEST DATA CENTER IEC
   *************  **   ~~~   **   This error occurred 4 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************
   *************  ** Warning ** CalcDoe2DXCoil: Coil:Cooling:DX:SingleSpeed="WEST DX COOLING COIL" - Air volume flow rate per watt of rated total cooling capacity is out of range error continues...
   *************  **   ~~~   **   This error occurred 125600 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=2.683985E-005  Min=1.081167E-005
   *************
   *************  ** Warning ** CalcDoe2DXCoil: Coil:Cooling:DX:SingleSpeed="WEST DX COOLING COIL" - Low condenser dry-bulb temperature error continues...
   *************  **   ~~~   **   This error occurred 3 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=-0.15 [C]  Min=-1.375 [C]
   *************
   *************  ** Warning ** CalcDoe2DXCoil: Coil:Cooling:DX:SingleSpeed="WEST DX COOLING COIL" - Full load outlet temperature indicates a possibility of frost/freeze error continues. Outlet air temperature statistics follow:
   *************  **   ~~~   **   This error occurred 45192 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=1.999938  Min=-48.246852
   *************
   *************  ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Tower range temperature is out of range error continues...
   *************  **   ~~~   **   This error occurred 18760 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=1.110626  Min=1.215599E-005
   *************
   *************  ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Tower approach temperature is out of range error continues...
   *************  **   ~~~   **   This error occurred 135282 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=1.10822  Min=0.
   *************
   *************  ** Warning ** CoolingTower:VariableSpeed "COOLING TOWER" - Inlet air wet-bulb temperature is out of range error continues...
   *************  **   ~~~   **   This error occurred 34 total times;
   *************  **   ~~~   **   during Warmup 0 times;
   *************  **   ~~~   **   during Sizing 0 times.
   *************  **   ~~~   **   Max=26.940529  Min=26.679421
   *************
   ************* EnergyPlus Warmup Error Summary. During Warmup: 0 Warning; 0 Severe Errors.
   ************* EnergyPlus Sizing Error Summary. During Sizing: 4 Warning; 0 Severe Errors.
   ************* EnergyPlus Completed Successfully-- 324890 Warning; 2 Severe Errors; Elapsed Time=00hr 07min 54.86sec`

I am not certain if the problem is the Ubuntu update or something else. 

https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes/ChangeSummary/18.04.2

**Comment by myndtrust** [comment]
Update: Assuming that the issues was with the Ubuntu update, I deployed on GCP with a 16.04 Ubuntu VM. The problem persisted, so this is not an OS issue.

I then commented out the single weather epw (Chicago) file and un-commented the the comma separated set of weather files in the .bashrc.
 
Run appears to be working - but this root cause is puzzling. The IDF has Chicago noted, the README notes the San Francisco, and in practice this only works with the the set of weather files. I am doing this right?
